charts/newrelic-logging/README.md

```diff
@@ -14,14 +14,14 @@ This document explains how to install it in your cluster, either using a [Helm](
 
  2. Add the New Relic official Helm chart repository following [these instructions](../../README.md#installing-charts)
 
- 3. Run the following command to install the New Relic Logging Kubernetes plugin via Helm, replacing the placeholder value `YOUR_LICENSE_KEY` with your [New Relic license key](https://docs.newrelic.com/docs/accounts/install-new-relic/account-setup/license-key):
+ 3. Run the following command to install the New Relic Logging Kubernetes plugin via Helm, replacing the placeholder value `YOUR_LICENSE_KEY` with your [New Relic license key](https://docs.newrelic.com/docs/accounts/install-new-relic/account-setup/license-key) and `YOUR_CLUSTER_NAME` with your cluster name:
     * Helm 3
         ```sh
-        helm install newrelic-logging newrelic/newrelic-logging --set licenseKey=YOUR_LICENSE_KEY
+        helm install newrelic-logging newrelic/newrelic-logging --set licenseKey=YOUR_LICENSE_KEY --set cluster=YOUR_CLUSTER_NAME
```

Comment:
The `cluster` parameter is technically not required for the Logs integration if you don't want to decorate your logs with a `cluster_name`, however, most people will want to know what cluster their logs are associated with. 
 I would argue strongly that it makes sense to make it a required field.
 
 The variable originates [here]( https://github.com/newrelic/helm-charts/blob/master/charts/newrelic-logging/templates/_helpers.tpl#L76).  It is set as an ENV variable in the Daemonset [here](https://github.com/newrelic/helm-charts/blob/master/charts/newrelic-logging/templates/daemonset.yaml#L65) so that it can be referenced in the fluent-bit filter [here](https://github.com/newrelic/helm-charts/blob/master/charts/newrelic-logging/templates/configmap.yaml#L38).
 
 Thoughts?

Reply:
@jsubirat see above